cmd/gateway-gcs.go

```diff
@@ -459,11 +458,14 @@ func (l *gcsGateway) GetObjectInfo(bucket string, object string) (ObjectInfo, er
 	}
 
 	attrs, err := l.client.Bucket(bucket).Object(object).Attrs(l.ctx)
+
 	if err != nil {
 		return ObjectInfo{}, gcsToObjectError(traceError(err), bucket, object)
 	}
+	objInfo := fromGCSObjectInfo(attrs)
+	objInfo.ETag = fmt.Sprintf("%d", attrs.CRC32C)
```

Comment:
I think the function naming is correct, isn't it? It is constructed from GCS attributes - maybe fromGCSToObjectInfo?